metadata/src/main/java/org/apache/kafka/image/MetadataDelta.java

```diff
@@ -366,7 +367,7 @@ public MetadataImage apply() {
     @Override
     public String toString() {
         return "MetadataDelta(" +
-            "highestOffset=" + highestOffset +
+            "highestOffset=" + highestOffsetExclusive +
```

Comment:
Let's change the string literal `"highestOffset="` to match the internal field of this type. E.g. `"highestOffsetExclusive="`.